### PR TITLE
Fix extra port icon

### PR DESCRIPTION
**Pull request by alexr00**

### CODE REVIEW

**Comment by alexr00** [comment]
The `Add` row of the the Ports view shouldn't have a label or icon in any of the columns. This PR removes both the label and icon from the Privacy column of the Add row.

File: src/vs/workbench/contrib/remote/browser/tunnelView.ts [hunk 1]
```diff
@@ -268,9 +268,15 @@ class PrivacyColumn implements ITableColumn<ITunnelItem, ActionBarCell> {
 				['tunnelType', row.tunnelType],
 				['tunnelCloseable', row.closeable]
 			];
-		const label = row.privacy === TunnelPrivacy.Public ? nls.localize('tunnel.privacyPublic', "Public") : nls.localize('tunnel.privacyPrivate', "Private");
+		let label: string = '';
```
Comment (sandy081) [inline src/vs/workbench/contrib/remote/browser/tunnelView.ts:271]: Do early return if tunnel type is `add`

File: src/vs/workbench/contrib/remote/browser/tunnelView.ts [hunk 2]
```diff
@@ -546,8 +552,13 @@ class TunnelItem implements ITunnelItem {
 	get icon(): ThemeIcon | undefined {
 		switch (this.privacy) {
 			case TunnelPrivacy.Public: return publicPortIcon;
-			default: return privatePortIcon;
-
+			default: {
```
Comment (sandy081) [inline src/vs/workbench/contrib/remote/browser/tunnelView.ts:555]: Also same here - Do early return if tunnel type is `add`

**Review by sandy081 — CHANGES_REQUESTED** [review]

**Review by sandy081 — APPROVED** [review]
Thanks for the changes. Approved.

**Pull request merged**